core/storage/question/gae_models.py

```diff
@@ -218,6 +218,10 @@ def get_question_skill_links_and_skill_descriptions(
                 used for the next page (or None if no more pages are left). The
                 returned next cursor value is urlsafe.
         """
+        question_skill_count = min(
```

Comment:
Test this in gae_models.test
(i.e. test a case when len(skill_ids) > constants. MAX_SKILLS_PER_QUESTION
and when
len(skill_ids) < MAX_SKILLS_PER_QUESTION

Reply:
Done.